app/reducers/app/index.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import build from './build';
+import version from './version';
+
+export default combineReducers({
```

Comment:
~~What are these used for? They aren't set any where~~

Oh, they're set by the middleware

Reply:
Yeap